inc/patterns/footer-dark.php

```diff
@@ -6,8 +6,8 @@
 	'title'      => __( 'Dark footer with title and citation', 'twentytwentytwo' ),
 	'categories' => array( 'footer' ),
 	'blockTypes' => array( 'core/template-part/footer' ),
-	'content'    => '<!-- wp:group {"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|background"}}},"spacing":{"padding":{"top":"max(1.25rem, 5vw)","bottom":"max(1.25rem, 5vw)","left":"max(1.25rem, 5vw)","right":"max(1.25rem, 5vw)"}}},"backgroundColor":"foreground","textColor":"background","layout":{"inherit":true}} -->
-					<div class="wp-block-group alignfull has-background-color has-foreground-background-color has-text-color has-background has-link-color" style="padding-top:max(1.25rem, 5vw);padding-right:max(1.25rem, 5vw);padding-bottom:max(1.25rem, 5vw);padding-left:max(1.25rem, 5vw)"><!-- wp:group {"align":"wide","layout":{"type":"flex","justifyContent":"space-between"}} -->
+	'content'    => '<!-- wp:group {"align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|background"}}},"spacing":{"padding":{"top":"max(1.25rem, 5vw)","bottom":"max(1.25rem, 5vw)"}}},"backgroundColor":"foreground","textColor":"background","layout":{"inherit":true}} -->
```

Comment:
From my reading of #309, it sounded like we should actually be using the `var(--wp--custom--spacing--small, 1.25rem)` everywhere, right? So it can pull from the variable? That generally makes sense, but I'm confused about why you're using a hardcoded value here instead?

Reply:
I wanted this PR to remove the left/right padding - which makes the outer padding work better with the existing patterns. 

I didn’t want to change too much at once - but yea those should be using variables as well.